frontend-react/src/config/endpoints/lookupTables.ts

```diff
@@ -0,0 +1,63 @@
+import { HTTPMethods, RSEndpoint } from ".";
+
+// the shape used by the frontend client for value sets
+export interface ValueSet {
+    name: string;
+    createdBy: string;
+    createdAt: string;
+    system: string;
+}
+
+// the shape sent down by the API for value sets
+export interface ApiValueSet {
+    name: string;
+    created_by: string; // unused
+    created_at: string; // unused
+    system: string;
+    reference: string; // unused
+    referenceURL: string; // unused
+}
+
+export interface ValueSetRow {
+    name: string;
+    code: string;
+    display: string;
+    version: string;
+}
+
+export interface LookupTable {
+    lookupTableVersionId: number;
+    tableName: string;
+    tableVersion: number;
+    isActive: boolean;
+    createdBy: string;
+    createdAt: string;
+    tableSha256Checksum: string;
+}
+
+export enum LookupTables {
+    VALUE_SET = "sender_automation_value_set",
+    VALUE_SET_ROW = "sender_automation_value_set_row",
+}
+
+// notice the react-router style colon demarcated dynamic path segments
```

Comment:
Can you add JSDOCs outlining the endpoints available from this object?
See: https://github.com/CDCgov/prime-reportstream/blob/cceb4f0306756c55ea600acaa6fe1f36f2cea968/frontend-react/src/network/api/History/Reports.ts#L70-L74